metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/data_resolver.py

```diff
@@ -97,6 +99,8 @@ def __init__(
             ),
         )
 
+        self.get_app = lru_cache(maxsize=128)(self.__get_app)
```

Comment:
why would we be calling get_app multiple times?

Reply:
Not needed, I removed the cache.